.rubocop.yml

```diff
@@ -1,11 +1,80 @@
-require: rubocop-rails
+require:
+  - rubocop-rails
+  - rubocop-rspec
 
 AllCops:
+  TargetRubyVersion: 3.0.2
+  NewCops: enable
   Exclude:
-    - 'spec/**/*'
-    - 'db/migrate/*'
-    - 'bin/*'
-    - 'db/schema.rb'
-    - 'config/initializers/devise.rb'
-    - vendor/**/*.rb 
-    - vendor/bundle/**/*
\ No newline at end of file
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'vendor/**/*'
+    - 'script/**/*'
+    - 'bin/**/*'
+    - 'node_modules/**/*'
+    - 'spec/vcr/**/*'
+
+RSpec/NestedGroups:
+  Max: 6
+
+RSpec/ImplicitSubject:
+  Enabled: false
+
+RSpec/MultipleExpectations:
+  Enabled: false
+
+Rails/LinkToBlank:
+  Enabled: false
+
+RSpec/ExampleLength:
+  Enabled: false
+
+RSpec/DescribedClass:
+  Enabled: false
+
+RSpec/DescribeClass:
+  Enabled: false
+
+Naming/VariableNumber:
+  Exclude:
+    - 'spec/**/*_spec.rb'
+
+RSpec/Capybara:
+  Enabled: false
+
+Metrics/BlockLength:
+  Exclude:
+    - 'Rakefile'
+    - '**/*.rake'
+    - 'spec/**/*_spec.rb'
+    - 'spec/support/**/*'
+    - 'spec/factories/*'
+    - 'config/**/*.rb'
+    - 'app/admin/**/*.rb'
+    - 'app/views/admin/**/*.arb'
```

Comment:
You can probably remove those since we don't have an admin panel yet.
I know they were present in the template I sent you - sorry, I must have missed them